packages/versioning/src/lib.ts

```diff
@@ -51,6 +51,12 @@ const libDef = {
         default: "@renamedFrom.oldName cannot be empty string.",
       },
     },
+    "incompatible-decorator-use": {
+      severity: "error",
+      messages: {
+        default: paramMessage`Namespace '${"name"}' cannot be decorated with @versioned and @service({version: ${"version"}}). Remove the version argument from @service.`,
```

Comment:
```suggestion
        default: paramMessage`Namespace '${"name"}' specify a fixed service version with @service({version: ${"version"}}) while using `@versioned`. Remove the version argument from @service.`,
```
maybe something like that to be more clear that the problem is `version:` not the `@service` decorator